apps/vaporgui/CMakeLists.txt

```diff
@@ -88,6 +88,47 @@ set (SRCS
 	Histo.h
 	EventRouter.h
 	GLWidget.h
+	VizWin.h
+	Core3_2_context.h
+	TrackBall.h
+	Manip.h
+	NavigationEventRouter.h
+	AnimationEventRouter.h
+	AnimationParams.h
+	AnnotationEventRouter.h
+	BannerGUI.h
+	BarbEventRouter.h
+	BarbEventRouter.h
+	Combo.h
+	ContourEventRouter.h
+	ControlPointEditor.h
+	CopyRegionWidget.h
+	DomainWidget.h
+	FileOperationChecker.h
+	GLColorbarWidget.h
+	GUIStateParams.h
+	GeometryWidget.h
+	ImageEventRouter.h
+	ImageEventRouter.h
+	ImageFrame.h
+	ImageFrame.h
+	IsoSurfaceEventRouter.h
+	MappingFrame.h
+	MouseModeParams.h
+	PlotParams.h
+	QIntValidatorWithFixup.h
+	RangeCombos.h
+	RangeController.h
+	RenderEventRouter.h
+	RenderHolder.h
+	SettingsEventRouter.h
+	SettingsParams.h
+	StatisticsParams.h
+	TFWidget.h
+	TabManager.h
+	VariablesWidget.h
+	VizSelectCombo.h
+	WireFrameEventRouter.h
```

Comment:
What difference will these changes make to cmake? VAPOR seemed to build well before these headers are added.

Reply:
It makes CMake aware of these headers. It doesn't affect the build but it is useful when generating files for Visual Studio and XCode.

Reply:
Interesting. Does this mean that we should make it a standard practice to include header files in the CMakeList.txt in our future development?

Reply:
Its not required but I would encourage it